features/karaf/src/main/feature/feature.xml

```diff
@@ -41,6 +41,7 @@
 
     <bundle start-level="75">mvn:org.eclipse.smarthome.core/org.eclipse.smarthome.core.thing.xml/${project.version}</bundle>
     <bundle>mvn:org.eclipse.smarthome.core/org.eclipse.smarthome.core.transform/${project.version}</bundle>
+    <bundle>mvn:org.eclipse.smarthome.core/org.eclipse.smarthome.core.audio/${project.version}</bundle>
```

Comment:
It does need to be target in this PR, but is voice and audio really a base part that needs to be available?
Isn't that some optional feature that could be added...